src/basilisp/lang/map.py

```diff
@@ -210,8 +212,15 @@ def _lrepr_py_dict(o: dict, **kwargs: Unpack[PrintSettings]) -> str:
     return f"#py {map_lrepr(o.items, '{', '}', **kwargs)}"
 
 
+reduce_init_obj = object()
```

Comment:
This appears to be unused.

Reply:
Good catch. Leftover from when I implemented reduce on Map (which I don't think is necessary).